docs/proposals/reliable-message-delivery.md

```diff
@@ -216,6 +216,15 @@ type ObjectSyncStatus struct {
 }
 ```
 
+#### ObjectSync CRD Garbage Collection
```

Comment:
We can move it to the `Exception scenarios/Corner cases handling` section below

Reply:
@fisherxu 
ok, Done.